.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.9.1",
-  "packages/api-explorer": "0.9.11",
-  "packages/code-editor": "0.1.4",
-  "packages/extension-api-explorer": "21.8.1",
+  ".": "1.9.2",
+  "packages/api-explorer": "0.9.12",
+  "packages/code-editor": "0.1.5",
+  "packages/extension-api-explorer": "21.8.2",
   "packages/extension-sdk": "21.8.2",
   "packages/extension-sdk-react": "21.8.2",
   "packages/hackathon": "21.8.1",
-  "packages/run-it": "0.9.11",
+  "packages/run-it": "0.9.12",
   "packages/sdk": "21.8.2",
-  "packages/sdk-codegen": "21.0.19",
-  "packages/sdk-codegen-scripts": "21.0.20",
+  "packages/sdk-codegen": "21.0.20",
+  "packages/sdk-codegen-scripts": "21.0.21",
```

Comment:
@jkaster FYI all of these bumps are ripple effect from the change I made to `python.gen.ts` for the python sdk bugfix